f2/src/forms/PrivacyConsentInfoForm.js

```diff
@@ -57,15 +57,13 @@ export const PrivacyConsentInfoForm = props => {
             shouldWrapChildren
             spacing={6}
           >
-            {submitFailed && hasValidationErrors ? (
-              <ErrorSummary onSubmit={handleSubmit} errors={errors} />
-            ) : null}
+            {submitFailed && hasValidationErrors ? <ErrorSummary /> : null}
 
```

Comment:
Why don’t you pass on submit and errors anymore?

Reply:
It appeared that we did not need them from the beginning...

I've tried removing them and things just worked. Is it possible that ErrorSummary inherits props from the Form component?